plugins/hls-brittany-plugin/hls-brittany-plugin.cabal

```diff
@@ -0,0 +1,69 @@
+cabal-version: 2.2
+name:          hls-brittany-plugin
+version:       1.0.0.0
+synopsis:      Integration with the Brittany code formatter
+license:       Apache-2.0
+license-file:  LICENSE
+author:        The Haskell IDE Team
+copyright:     The Haskell IDE Team
+maintainer:    alan.zimm@gmail.com
+category:      Development
+build-type:    Simple
+extra-source-files:
+  LICENSE
+
+library
+  exposed-modules:  Ide.Plugin.Brittany
+  hs-source-dirs:   src
+  build-depends:    base
+               ,    brittany      >= 0.13.1.0
+               ,    filepath
+               ,    ghc
+               ,    ghc-boot-th
+               ,    ghcide
+               ,    lsp-types
+               ,    hls-plugin-api
+               ,    lens
+               ,    text
+               ,    transformers
+
+  default-language: Haskell2010
+
+executable test-server
+  default-language:   Haskell2010
+  build-depends:
+    , base
+    , data-default
+    , ghcide
+    , hls-brittany-plugin
+    , hls-plugin-api
+    , shake
+  main-is: Server.hs
+  hs-source-dirs: test
+  ghc-options: -threaded
+
+test-suite tests
+  type:               exitcode-stdio-1.0
+  default-language:   Haskell2010
+  build-tool-depends:
+    hls-brittany-plugin:test-server -any,
+  hs-source-dirs: test
+  main-is: Main.hs
+  build-depends:
+    , aeson
+    , base
+    , bytestring
+    , data-default
+    , deepseq
+    , ghcide                >= 0.7.5.0
```

Comment:
It should be `^>= 1.0.0.0`, right?

Reply:
I'm not convinced putting such strong upper bounds in the plugins is a good idea. It will be come clear in the long run, when the burden of releasing or relaxing those upper bounds becomes unsustainable